controller/bin/boot

```diff
@@ -26,9 +26,10 @@ sleep $(($ETCD_TTL+1))
 
 function etcd_set_default {
 	set +e
-	etcdctl --no-sync -C $ETCD mk $ETCD_PATH/$1 $2 >/dev/null 2>&1
-	if [[ $? -ne 0 && $? -ne 4 ]]; then
-		echo "etcd_set_default: an etcd error occurred. aborting..."
+	ERROR="$(etcdctl --no-sync -C $ETCD mk $ETCD_PATH/$1 $2 2>&1 >/dev/null)"
```

Comment:
so you only want stderr in ERROR? Why not `ERROR="$(etcdctl --no-sync -C $ETCD mk $ETCD_PATH/$1 $2 >/dev/null)"`, this should send stdout to `/dev/null`